core/tests/test_metrics_view.py

```
@@ -21,151 +24,99 @@
 from core.urls import metrics_view
 
 PROMETHEUS_200_PATCH = "core.urls.ExportToDjangoView"
+VALID_TOKEN = "super-secret-scrape-token-abc123"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "VALID_TOKEN"

(S105)